ember-app/app/components/columns/hb-column.js

```diff
@@ -113,7 +113,7 @@ var HbColumnComponent = Ember.Component.extend(SortableMixin, ScrollingColumn, {
 
     var issuesLength = this.get('visibleIssues').length;
     if(horizonVisible && issuesLength < totalColumnLength){
-      Ember.run.debounce(this, 'revealIssues' , horizon, 250);
+      Ember.run.debounce(this, 'revealIssues' , horizon, 200);
```

Comment:
I though about that too, its worth experimenting with. My concern with is the first one fires immediately, but the scroll is so fast the next one is throttled and the user will have to scroll back up and down again to trigger it.